### PR TITLE
chore(travis): Change to numberOfJobs

### PR DESCRIPTION
**Pull request by jervi**
Depends on https://github.com/spinnaker/halyard/pull/1355, should be merged when this is released.

### CODE REVIEW

File: setup/ci/travis.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ name):
    --address https://api.travis-ci.org \
    --base-url https://travis-ci.org \
    --github-token <token> \ # The GitHub token to authenticate to Travis
-   --number-of-repositories # How many repos the integration should fetch each
-                            # time the poller runs, higher than max expected
-                            # during polling interval
+   --number-of-jobs # How many jobs the integration should fetch each
+                    # time the poller runs, higher than max expected
```
Comment (dorbin) [inline setup/ci/travis.md]:
```suggestion
                    # time the poller runs. Set this higher than the max
```

File: setup/ci/travis.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ name):
...
+                    # during polling interval. Defaults to 100.
```
Comment (dorbin) [inline setup/ci/travis.md]:
```suggestion
                    # you expect during polling interval.
                    # Defaults to 100.
```

**Review by dorbin — COMMENTED** [review]
@jervi 
I've suggested a small change the wording (the old wording that was already there). 
Can you assign or tag whoever winds up reviewing the hal PR?
Thanks!

**Comment by jervi** [comment]
https://github.com/spinnaker/halyard/pull/1717 is now merged, so this can be merged once that is released. @ezimanyi

**Review by ezimanyi — APPROVED** [review]

**Review by dorbin — APPROVED** [review]

**Comment by dorbin** [comment]
Gonna wait and merge this concurrent-ish with the next Halyard release.

**Comment by 317brian** [comment]
Was there anything else that was needed @dorbin? Halyard 1.37 has the commit: https://github.com/spinnaker/halyard/compare/version-1.36.0...version-1.37.0

**Comment by ezimanyi** [comment]
@Mergifyio refresh

**Pull request merged**